spec/controllers/study_searches_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe StudySearchesController, type: :controller do
+    
+    describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "POST #create" do
+    it "returns http found" do
+     study_search_test = FactoryBot.attributes_for(:core_study_search)
+      post :create, core_study_search:study_search_test
```

Comment:
`post :create, core_study_search: study_search_test`

If you don't add that space it looks like `Core::StudySearch`